public/wp-content/plugins/attendance-plugin/attendance.php

```diff
@@ -1,38 +1,267 @@
-        wp_enqueue_script('cc-bootstrap-tether','https://cdnjs.cloudflare.com/ajax/libs/tether/1.4.0/js/tether.min.js');
-        wp_enqueue_script('cc-fontawesome-icons','https://use.fontawesome.com/ffc2c94a85.js');
-   }
- function cc_plugin_styles($hook) {
- 		if($hook != 'toplevel_page_attendance') {
-                return;
-        } 
-        wp_enqueue_style('cc-bootstrap4-style', get_template_directory_uri().'/dist/lib/css/bootstrap4.min.css');
-        wp_enqueue_style('cc-fonts','https://fonts.googleapis.com/css?family=Oswald|Marcellus+SC|Roboto|Open+Sans');
-        wp_enqueue_style('style', get_template_directory_uri().'/style.css');
-   }
- 
+			return;
+		}
+		wp_enqueue_script('cc-bootstrap4-script', get_template_directory_uri().'/dist/lib/js/bootstrap4.min.js', array('jquery'), '1.0.0', true);
+		wp_enqueue_script('cc-bootstrap-tether','https://cdnjs.cloudflare.com/ajax/libs/tether/1.4.0/js/tether.min.js');
+		wp_enqueue_script('cc-fontawesome-icons','https://use.fontawesome.com/ffc2c94a85.js');
+		wp_enqueue_script('main', get_template_directory_uri().'/main.js', array('jquery'), '1.0.0', true);
+		wp_localize_script( 'main', 'PARAMS', array('ajaxurl' => admin_url('admin-ajax.php')) );
+	}
+	
+	function cc_plugin_styles($hook){
+		if($hook != 'toplevel_page_attendance'){
+			return;
+		} 
+		wp_enqueue_style('cc-bootstrap4-style', get_template_directory_uri().'/dist/lib/css/bootstrap4.min.css');
+		wp_enqueue_style('cc-fonts','https://fonts.googleapis.com/css?family=Oswald|Marcellus+SC|Roboto|Open+Sans');
+		wp_enqueue_style('style', get_template_directory_uri().'/style.css');
+	}
+	 
+	function show_guest_tables(){
+		if(isset($_POST['attendance_event_id'])){
+			$event_id=$_POST['attendance_event_id'];
+			$invited_guest_table=invited_guest_body($event_id);
+			$guest_requests_table=requested_guest_body($event_id);
+			echo $invited_guest_table;
+			echo $guest_requests_table;
+		}
+		wp_die();
+	}
+	add_action('wp_ajax_show_guest_tables','show_guest_tables');
+	add_action('wp_ajax_nopriv_show_guest_tables','show_guest_tables');
+
+	function show_approved_guests(){
+		{
+			$all_guest_post_args=array(
+					'post_type' => 'guest',
+					'category_name' => 'approved'
+				);
+			$all_guest_query = new WP_Query($all_guest_post_args);
+			$table_allguests_header='';
+			$table_allguests_body='';
+			$table_allguests_footer='';
+			$table_allguests_header.='<div class="row">
+										<div class="col-lg-12 col-md-12 col-sm-12">
+											<div class="table-headings">Your Guest List</div><br>
+												<table class="table">
+													<thead>
+														<tr>
+															<th>Name</th>
+															<th>Email</th>
+															<th>Phone No.</th>
+															<th>Gender</th>
+															<th>Category</th>
+														</tr>
+													</thead>
+													<tbody>';
```

Comment:
your header and footer are not always changing right? In that case rather than loading the complete table, you can just request for <tbody> HTML.